src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs

```diff
@@ -272,6 +278,16 @@ private void DisplayInlineTypeHints_Unchecked(object sender, RoutedEventArgs e)
             UpdateInlineHintsOptions();
         }
 
+        private void RunCodeAnalysisInSeparateProcess_Checked(object sender, RoutedEventArgs e)
```

Comment:
Does this fire only if the user changes it or if it's changed programmatically? The useful test would be to turn it off, restart VS, and see if we disable it when the options page loads the first time.